assets/src/components/detours/diversionPage.tsx

```diff
@@ -144,6 +146,7 @@ export const DiversionPage = ({
             startPoint={startPoint ?? undefined}
             endPoint={endPoint ?? undefined}
             waypoints={waypoints}
+            unfinishedRouteSegments={unfinishedRouteSegments}
```

Comment:
This needs to be gated behind a test group, so that's why this PR is still in Draft. 

I think it's sort of ready for review though!